src/core/lender/tranche/test/tranche.t.sol

```diff
@@ -0,0 +1,93 @@
+// Copyright (C) 2020 Centrifuge
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+pragma solidity >=0.4.23;
+
+import "ds-test/test.sol";
+
+import "../tranche.sol";
+import "../../../test/simple/token.sol";
+
+contract Hevm {
+    function warp(uint256) public;
+}
+
+contract TrancheTest is DSTest {
+    Tranche tranche;
+    address tranche_;
+    SimpleToken token;
+    SimpleToken currency;
+
+    address self;
+
+    function setUp() public {
+        // Simple ERC20
+        token = new SimpleToken("TIN", "Tranche", "1", 0);
```

Comment:
Can we tinlake-erc20 for this instead?

Reply:
`SimpleToken` is only for testing. No `auth` on the mint method to make it easier